go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/operator-framework/operator-lib
 
-go 1.21
+go 1.22.0
 
 require (
 	github.com/go-logr/logr v1.4.1
 	github.com/onsi/ginkgo/v2 v2.17.1
 	github.com/onsi/gomega v1.32.0
-	github.com/operator-framework/api v0.23.0
+	github.com/operator-framework/api v0.24.0
 	github.com/prometheus/client_golang v1.19.0
 	github.com/prometheus/client_model v0.6.1
-	k8s.io/api v0.29.2
-	k8s.io/apimachinery v0.29.2
-	k8s.io/client-go v0.29.2
+	k8s.io/api v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
```

Comment:
I _think_ I need to update this still

Reply:
https://github.com/operator-framework/operator-lib/pull/176/commits/662cd2c7843ec5bb902a4eaa77e924e725cc28d8